js/data/symbol_bucket.js

```diff
@@ -404,48 +488,19 @@ SymbolBucket.prototype.addToDebugBuffers = function(collisionTile) {
                 var maxZoom = Math.max(0, Math.min(25, this.zoom + Math.log(box.maxScale) / Math.LN2));
                 var placementZoom = Math.max(0, Math.min(25, this.zoom + Math.log(box.placementScale) / Math.LN2));
 
-                addCollisionVertex(buffer, anchorPoint, tl, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, tr, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, tr, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, br, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, br, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, bl, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, bl, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, tl, maxZoom, placementZoom);
-
-                this.elementGroups.collisionBox.current.vertexLength += 8;
+                this.add.collisionBox.vertex(anchorPoint, tl, maxZoom, placementZoom);
```

Comment:
I think we should define `addVertex` local variable for calls like this to make them look less confusing. Just like you did in addSymbols, but everywhere. It may improve perf too.